src/mgmt/tests/srv_drpc_tests.c

```diff
@@ -52,6 +52,7 @@
 static const char	*TEST_ACES[] = {"A::OWNER@:rw",
 					"A::niceuser@:rw",
 					"A:G:GROUP@:r"};
+static uint32_t TEST_IDXS[] = {0,1,2};
```

Comment:
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)